AirCasting/Dashboard/Reordering/CustomSwipeAction.swift

```diff
@@ -0,0 +1,79 @@
+// Created by Lunar on 29/08/2022.
+//
+
+import Foundation
+import SwiftUI
+
+// Adds custom swipe action to a given view
```

Comment:
you can use a triple `/` to make a documentation out of this comment (the benefit of a documentation comment is that it will pop up whenever the user alt-clicks to reveal quick help. For example:
![image](https://user-images.githubusercontent.com/10663361/187678721-5c04f322-426a-45e4-8ab3-2ece23e7e020.png)

Which I think is quite helpful. We can also build the whole projects documentation and have it as a reference in XCode just like the standard SDKs have (or in the future export it to a webpage/wiki/etc).

I think that's a great value for just one more slash :)